### ISSUE TITLE
the human mind will always be superior to machines because machines are only tools to human minds

### ISSUE THREAD

**Issue by lihaoyuq** [lihaoyuq/1#1]
the statement of human mind is always superior to machines is too assertive and superficial . the comparison depends on which aspects we consider.
  We human beings invite and produce machine to assist people to complete many complex problems , so it is convincing that human superior to machines in intelligence. No matter how advancement the machine is , it is create and governed by human . nothing can be more superior than our intelligence, which we can use them to discover the mistery of space ,the mechanism of how the world operate, as well as the revolution of creatures in nature. We can make use of what we know and create machines to better service us .At present we can even travel to the mars .all of these above cannot be done by machines .
  In the other aspect ,however ,machines can also do a lot of things that we cannot do . they can working day and night without getting tired , no matter how long they works ,their working efficiency will  never decline . meanwhile ,they will never be affected by mood like human, because they have no motion at all. By contrast ,when we are depressed by difficulties or people in our life ,our work will be disturb too. Machines are working by the programs setting by people , and their working efficiency are better than human . besides ,they can also finish extremly difficult problems which we cannot slove. For example, super computers developed by scientists can calculate complex equations in a second, and the same question will take us human like several months. and computers are more precise . in these aspects , machines are superior than human .
  In fact, whether human are superior to machine depends on which criterion we use . Human are smarter than machine , we have creative thinking and we can make innovation. Machines can work more effectively and precisely. These are two important factors in our society . the deficiency of which can cause serious problems. we shouldn’t regard machine just as our tools in solving problems , instead they are our partners to cooperate well with human intelligence to make our society better operate.
  For example , when people commit crimes in a city , electric supervise devises can record what happened at any cornor and at anytime in the street or within buildings . polices ,by analysing the record , finding clues to catch the offenders . We can conclude that electric supervise devises are better than police because police cannot record any details happened outsides by using their eyes . but without police , criminals will never be catches and the safety of our society cannot be assured . within this idea police are superior than supervise devises.  This is a small examples of comparison of human and machines .different views will get different results . just as when we compare a flower and a tree , trees are taller and flowers are more colorful .
  To sum up ,we should have equal sights on human and machines .
